assessment.md

```diff
@@ -201,8 +201,8 @@ TODO: add description for each assessment level for this skill.
 
 * :flushed: = Does not have the tools installed to run the software in the first place.
 * :worried: = Has the correct tools installed but does not run the software or test cases.
-* :neutral_face: = Has the correct tools installed and runs the software, but doesn't run the test cases.
-* :blush: = Has the correct tools installed, runs the software, and runs the test cases.
+* :neutral_face: = Has the correct tools installed and runs the software, but doesn't run effective test cases.
+* :blush: = Has the correct tools installed, runs the software, and runs effective test cases.
 * :laughing: = identifies and understands which tools to use to test the software while
     also providing documentation and advice that helps other team members develop
     this skill.
```

Comment:
There are still no mentions of **Writing** comprehensive test cases here which is an important skill in the file of software engineering.